docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -34,43 +34,47 @@ By default, columns are ordered according to the order they are included in the
 > Otherwise, you take the risk of losing the column width state (if resized).
 > You can create the array outside of the render function or memoize it.
 
-## Column headers
+### Title & Description
 
 You can configure the headers with:
 
 - `headerName`: The title of the column rendered in the column header cell.
 - `description`: The description of the column rendered as tooltip if the column header name is not fully displayed.
-- `hide`: Hide the column.
-
-{{"demo": "pages/components/data-grid/columns/HeaderColumnsGrid.js", "bg": "inline"}}
 
-For more advanced header configuration, go to the [rendering section](/components/data-grid/rendering/#header-cell).
+{{"demo": "pages/components/data-grid/columns/HeaderColumnsGrid.js", "bg": "inline", "defaultCodeOpen": false}}
 
-## Column width
+### Width
 
-By default, the columns have a width of 100 pixels.
-This is an arbitrary, easy to remember value.
-To change the width of a column, use the `width` property available in `GridColDef`.
+The default columns width is 100 pixels.
+- set the `width` property available in `GridColDef` to override that value.
 
-{{"demo": "pages/components/data-grid/columns/ColumnWidthGrid.js", "bg": "inline"}}
+{{"demo": "pages/components/data-grid/columns/ColumnWidthGrid.js", "bg": "inline",  "defaultCodeOpen": false}}
 
-### Fluid width
+#### Fluid width
```

Comment:
h4 aren't in the TOC. Considering the section is important, the only solution I can think of to make it easier to find is:

```suggestion
### Fluid width
```